search/search_repository.go

```diff
@@ -251,8 +260,8 @@ func generateSQLSearchInfo(keywords searchKeyword) (sqlParameter string) {
 
 // extracted this function from List() in order to close the rows object with "defer" for more readability
 // workaround for https://github.com/lib/pq/issues/81
-func (r *GormSearchRepository) search(ctx context.Context, sqlSearchQueryParameter string, start *int, limit *int) ([]models.WorkItem, uint64, error) {
-	db := r.db.Model(&models.WorkItem{}).Where("tsv @@ query")
+func (r *GormSearchRepository) search(ctx context.Context, sqlSearchQueryParameter string, workItemTypes []string, start *int, limit *int) ([]models.WorkItem, uint64, error) {
+	db := r.db.Debug().Model(models.WorkItem{}).Where("tsv @@ query")
```

Comment:
We should not have the `.Debug().` call in production I guess. Please remove it.